### ISSUE TITLE
Create more efficient patching practices for training data.

### ISSUE THREAD

**Issue by AndrewBeers** [QTIM-Lab/DeepNeuro#18]
Creating pre-patched datasets can take hours at certain sizes. Here are some possible solutions:

- Reimplement patch-extraction in Tensorflow, taking advantage of GPUs
- Write data in bulk, instead of one patch a time
